src/Chainweb/Rosetta/RestAPI/Server.hs

```diff
@@ -156,10 +157,18 @@ mempoolH
     -> Handler MempoolResp
 mempoolH v ms (MempoolReq net) = runExceptT work >>= either throwRosetta pure
   where
+    f :: TransactionHash -> TransactionId
+    f (TransactionHash h) = TransactionId (T.decodeUtf8 $ BSS.fromShort h)
+
     work = do
         cid <- validateNetwork v net
-        _ <- lookup cid ms ?? RosettaInvalidChain
-        error "not yet implemented"  -- TODO!
+        mp <- lookup cid ms ?? RosettaInvalidChain
+        r <- liftIO $ newIORef mempty
+        -- TODO: This will need to be revisited once we can add
+        -- pagination + streaming the mempool
+        _ <- liftIO $ mempoolGetPendingTransactions mp Nothing $ writeIORef r
```

Comment:
why are we ignoring these results?

Reply:
there's no useful result to ignore. `mempoolGetPendinTransactions` is

```haskell
Maybe HighwaterMark -> (Vector TransactionHash -> IO ()) -> IO HighwaterMark
```

We do not need the high water mark returned, just the vector of hashes in the pending pool. The only way to extract these hashes is via stateful callback (so we use an `IORef` to extract them).